packages/material-ui/src/styles/responsiveFontSizes.test.js

```diff
@@ -21,9 +22,11 @@ describe('responsiveFontSizes', () => {
     expect(typography.h1).to.deep.equal({
       ...defaultVariant,
       fontSize: '3.5rem',
-      '@media (min-width:600px)': { fontSize: '4.75rem' },
-      '@media (min-width:960px)': { fontSize: '5.5rem' },
-      '@media (min-width:1280px)': { fontSize: defaultVariant.fontSize },
+      [`@media (min-width:${defaultTheme.breakpoints.values.sm}px)`]: { fontSize: '4.75rem' },
```

Comment:
My opinion is that the test should have hardcoded values. The main reason is, if we "break" the default theme, we won't notice it :)

Reply:
I think hardcoded values should be tested in one file which we already have in `createBreakpoints.test.js`. For `responsiveFontSizes.test.js`, we want to test that it use breakpoint from the theme, so variable make sense to me.